### ISSUE TITLE
An issue/bug in TestBench JUnit 5 documentation for Vaadin 23

### ISSUE THREAD

**Issue by bwajtr** [vaadin/docs#1986]
There is an issue in [this chapter](https://vaadin.com/docs/latest/testing/end-to-end/getting-started/#setting-up-your-project) of TestBench documentation. The code snippet there indicates that you should use "vaadin-testbench-junit5" artifact for JUnit 5 and that the version and scope of this artifact is predefined in Vaadin BOM. I have the following BOM imported in my project, but "vaadin-testbench-junit5" is not included there are at all (Vaadin version 23.3.1):
```
  <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>com.vaadin</groupId>
                <artifactId>vaadin-bom</artifactId>
                <version>${vaadin.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```
When I then try to use the following dependency in my project, as the docu states:
```
<dependency>
    <groupId>com.vaadin</groupId>
    <artifactId>vaadin-testbench-junit5</artifactId>
    <scope>test</scope>
</dependency>
```
then I get the following error:
```
Could not find artifact com.vaadin:vaadin-testbench-junit5:pom:unknown in central (https://repo.maven.apache.org/maven2)
```
I was not able to find any such artifact "vaadin-testbench-junit5" in maven central or anywhere... so unless it's present in some private Vaadin repo I think this is really a bug in documentation... 
Also, it's possible that the support for JUnit 5 will arrive in Vaadin 24, but if that is the case, then the documentation should say that or JUnit5 support should be removed from documentation for Vaadin 23

**Comment by jouni** [comment]
What’s the correct fix? Mark JUnit 5 to apply only for Vaadin 24? Or is it possible to use JUnit 5 with Vaadin 23.3? @mshabarov, can you tell?

**Comment by mshabarov** [comment]
Sorry for confusion. The support for JUnit is introduced in Vaadin 24.0 and Vaadin 23.3 doesn't have it. So `vaadin-testbench-junit5` is only available in Vaadin 24.0. I've made a patch to emphasise it in the docs and make it more visible.
